.packit.yaml

```
@@ -0,0 +1,40 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-foreman_scc_manager.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-foreman_scc_manager.spec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: foreman_scc_manager
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-foreman_scc_manager
+
+upstream_tag_template: v{version}
+
+actions:
+  post-upstream-clone:
+    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/plugins/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec -O rubygem-foreman_scc_manager.spec"
```

```suggestion
    - "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec -O rubygem-foreman_scc_manager.spec"
```

Your plugin lives in the "Katello" world, so it needs slight adjustments from the default template ;)

---

Right, thanks, I was going to ask that :+1: